src/collectors/ip/ip.py

```diff
@@ -103,7 +104,7 @@ def collect(self):
             data = data.split()
 
             # Zip up the keys and values
-            for i in xrange(1, len(header)):
```

Comment:
plz to use `six.moves.range` instead so we can preserve the same behavior on python2 and 3.